utils/publish/pre-publish-check-ci.js

```diff
@@ -0,0 +1,30 @@
+/* eslint-disable import/no-commonjs */
+/**
+ * Pre-publish checks to verify that our CI publish will go smoothly.
+ *
+ * NOTE: This script is run by changesets when the Release Manager is publishing
+ * a new version of the WB packages.
+ */
+const path = require("path");
+const glob = require("glob");
+
+const {
+    checkPublishConfig,
+    checkPackageMain,
+    checkPackageModule,
+    checkPackageSource,
+} = require("./pre-publish-utils.js");
+
+glob(
+    path.join(__dirname, "..", "..", "packages", "**", "package.json"),
+    (err, pkgPaths) => {
+        for (const pkgPath of pkgPaths) {
+            const pkgJson = require(path.relative(__dirname, pkgPath));
+
+            checkPublishConfig(pkgJson);
+            checkPackageMain(pkgJson);
+            checkPackageModule(pkgJson);
+            checkPackageSource(pkgJson);
+        }
```

Comment:
The main difference of this script is that we don't need to run `checkNpmUser` here because that's handled via the changesets action that uses a [npm automation token](https://github.blog/changelog/2020-10-02-npm-automation-tokens/).